openmc/mgxs/mgxs.py

```diff
@@ -1547,7 +1546,7 @@ def tallies(self):
         if self._tallies is None:
 
             # Create a list of scores for each Tally to be created
-            scores = ['flux', 'total', 'scatter-P1']
+            scores = ['flux', 'total', 'scatter-1']
```

Comment:
I *think* so... I mean starting from like the bell-Hansen-sandmeier derivation, the right hand side _should_ be a nu-scatter sigS, and therefore we would be subtracting off nu-scatter-1.